src/pages/test-report/components/examiner-takes-action/eta.html

```diff
@@ -1,16 +1,12 @@
 <ion-row>
-  <ion-col>
-    <button (click)=(toggleVerbal()) [ngClass]="{ complete: componentState.verbal$ | async }">
-      <div class="label">  
+  <ion-col class="only-right-padding  padding-right-10">
+    <button class="mes-test-report-button verbal-eta-button" (click)=(toggleVerbal()) [ngClass]="{ complete: componentState.verbal$ | async }">
```

Comment:
I know this syntax for binding to clicks works but can we stick to the official guidance from the angular docs and use `(click)="toggleVerbal()"`?

Reply:
Fixed all instances of it to the preferred way :)